File: Beginner/Is it a VOWEL or CONSONANT (VOWELTB)/vowel.java
import java.util.*;
import java.lang.*;
import java.io.*;


class Codechef {
	public static void main (String[] args) throws java.lang.Exception {
		Scanner scan=new Scanner(System.in);
		if(scan.hasNext()){
		    char c = scan.next().charAt(0);
		    if(c=='a'||c=='A'||c=='e'||c=='E'||c=='i'||c=='I'||c=='o'||c=='O'||c=='u'||c=='U')
		        System.out.println("Vowel");
		    else 
		        System.out.println("Consonant");
        }
        scan.close();
	}
}

File: Beginner/Is it a VOWEL or CONSONANT (VOWELTB)/vowel.cpp
#include<iostream>
using namespace std;
int main() {
    char ch;
    cin >> ch;
    if (ch =='A'|| ch =='E'|| ch =='I'|| ch =='O'|| ch =='U') {
        cout << "Vowel";
    }
    else {
        cout << "Consonant";
    }
    return 0;
}